bin/fixedcase/truelist

```diff
@@ -1,535 +1,3150 @@
+Anglesey; Sir Ynys Môn
 Anglo
 Angloromani
+Angola
+Angolar
+Angor
+Angoram
+Angosturas Tunebo
+Anguilla
+Anguthimri
+Anhui Sheng
+Ani Phowa
+Anibare
+Anii
 Animacy
+Animere
+Anindilyakwa
 Anita
+Anjam
+Ankara
+Ankave
+Anmatyerre
+Annaba
+Annobón
+Anong
+Anor
+Ansabā
+Anse Boileau
+Anse Etoile
+Anse Louis
+Anse Royale
+Anse aux Pins
+Anserma
+Ansus
+Antakarinya
+Antalya
+Antananarivo
+Antankarana Malagasy
+Antarctica
+Antigua and Barbuda
+Antigua and Barbuda Creole English
+Antioquia
+Antofagasta
+Antrim and Newtownabbey
+Antsiranana
 Anttila
+Antwerpen
+Anuak
+Anufo
+Anuki
+Anurādhapura
+Anuta
+Anyin
+Anyin Morofo
+Anzoátegui
+Ao Naga
+Aoheng
+Aomori
+Aore
+Aosta
+Aousserd
+Ap Ma
+Apac
 Apache
 Apache Solr
+Apalachee
+Apalaí
+Apali
+Apatani
+Apayao
+Apače
+Apes novads
+Apiaká
+Apinayé
+Apma
 Apollo
 Appendix A
+Appenzell Ausserrhoden
+Appenzell Innerrhoden
 April
+Aproumu Aizi
 Apsaalooke
+Apure
+Apurinã
+Apurímac
+Apurímac Quechua
+Aputai
+Aqmola oblysy
+Aqtöbe oblysy
 Aquinas
+Aquitanian
+Ar Raqqah
+Ar Rayyan
+Ar Riyāḍ
 Arab
+Arabana
 Arabela
+Arabian
 Arabic
+Arabic Sign Language
 Arabicized
 Arabish
 Arabizi
 Arabs
+Arad
+Aragacotn
 Aragonese
+Aragua
+Aragón
+Araki
+Arakwal
 Aramaic
+Arammba
+Aranadan
+Arandai
+Araona
 Arapaho
+Arapaso
+Ararandewára
+Ararat
+Arauca
+Araucanía
+Arawak
+Araweté
+Arawum
+Aračinovo
+Arbil
+Arbore
 Arbëreshë
 Archi
+Ardabīl
+Ardahan
+Ardennes
+Ardhamāgadhī Prākrit
+Ards and North Down
+Ardèche
+Areba
+Arem
+Arequipa
+Arezzo
 Argentina
+Argentine Sign Language
 Argentinian
+Argeș
+Argobba
+Argolida
 Argotario
+Arguni
+Argyll and Bute
+Arhangay
+Arhuaco
+Arhâ
+Arhö
+Ari
+Ariana
+Aribwatsa
+Aribwaung
+Arica y Parinacota
+Arigidi
+Arikapú
+Arikara
+Arikem
+Arima
 Arin
+Aringa
 Arizona
+Ariège
+Arkadia
 Arkansas
+Arkhangel'skaya oblast'
+Arma
+Armagh
+Armavir
+Armazic
+Armenia
 Armenian
+Armenian Sign Language
+Arno
 Aromanian
 Arora
+Arosi
 Arpitan
+Arrarnta
 Arrernte
+Arritinngithigh
+Arta
 Artetxe
+Artibonite
+Artigas
+Artvin
+Arua
+Aruamu
+Aruba
+Aruek
+Arunachal Pradesh
+Aruop
+Arusha
+Arutani
+Aruá
 Arvanitika
 Arxiv
 Aryan
+As Sulaymaniyah
+As Suwayda'
+As Suways
+As Sādis min Uktūbar
+Asaro'o
+Asas
+Ascoli Piceno
+Ash Shamal
+Ash Shamālīyah
+Ash Sharqīyah
+Ash Shāriqah
+Ashanti
+Ashe
 Asher
+Ashkun
+Asho Chin
+Ashtiani
+Asháninka
+Ashéninka Pajonal
+Ashéninka Perené
+Asia
 Asian
 Asiatic
+Asilulu
+Askopan
+Asmat
+Asoa
+Assaba
+Assam
 Assamese
+Assan
+Assangori
 Assiniboine
 Association for Computational Linguistics
 Assyriologists
+Astana
+Astara
+Asti
+Astrakhanskaya oblast'
 Asturian
+Asturias
+Asu
+Asue Awyu
+Asumboa
+Asunción
+Asunción Mixtepec Zapotec
+Asuri
+Aswān
+Asyūt
+At Ta'mim
+Ata
+Ata Manobo
+Atacama
+Atakapa
+Atakora
+Atampaya
+Atatláhuca Mixtec
+Atayal
+Atemble
+Athpariya
+Ati
 Atikamekw
 Atiya
+Atlantique
+Atlas Tamazight
+Atlántico
+Atlántico Norte
+Atlántico Sur
+Atlántida
+Atohwaim
+Atong
+Atorada
+Atsahuaca
+Atsam
+Atsugewi
+Attapady Kurumba
+Attapu
+Attard
+Attiki
+Attié
 Attorneys
```

Comment:
Why is Attorneys in the (previous) list?

Reply:
No idea